Gemfile.lock

```diff
@@ -154,7 +154,7 @@ GEM
     uglifier (2.7.2)
       execjs (>= 0.3.0)
       json (>= 1.8.0)
-    unicorn (4.9.0)
+    unicorn (5.0.1)
```

Comment:
This is the big update to gems.  Unicorn still crashes every 3 requests.  Something very bad is happening